add_varsel_cmap.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+#
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Modify an Emoji font to populate cmap 14 variation selector subtable.
+
+This uses the unicode variation selector data, finds all characters in the
+standard cmap that match, and generates a cmap format 14 subtable indicating
+that they have the provided presentation (emoji by default).  No fonts
+are processed if any already has a format 14 subtable.  Modified fonts
+are written to the provided dir and optionally have a provided suffix
+appended (before the extension).  An output file name can be provided
+in which case only one input file is allowed."""
+
+import argparse
+import os
+from os import path
+import re
+import sys
+
+from fontTools import ttLib
+from fontTools.ttLib.tables import _c_m_a_p as CMap
```

Comment:
The "as" part may be against Google python style (https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Imports#Imports), as we can't really say _c_m_a_p is inconveniently long.